graphql_compiler/schema_transformation/rename_schema.py

```diff
@@ -387,6 +387,10 @@ def _rename_and_suppress_types(
     """
     visitor = RenameSchemaTypesVisitor(renamings, query_type, scalars)
     renamed_schema_ast = visit(schema_ast, visitor)
+    if visitor.name_conflicts != {} or visitor.renamed_to_scalar_conflicts != {}:
```

Comment:
Can these be `None`?

Reply:
D'oh, I forgot to change these after #885 and it slipped my notice when I was re-reading yesterday.